modules/macos_utils/manifests/set_hostname.pp

```diff
@@ -0,0 +1,15 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+class macos_utils::set_hostname {
+
+    if $::operatingsystem == 'Darwin' {
+        exec { 'SetLocalHostName':
+            command => '/usr/sbin/scutil --set LocalHostName $(hostname|cut -d\. -f1)',
+            onlyif  => '/bin/test $(/usr/sbin/scutil --get LocalHostName) != "$(hostname|cut -d\. -f1)"',
+        }
+    } else {
+        fail("${module_name} does not support ${::operatingsystem}")
+    }
+}
```

Comment:
I don't think this should be set here.  Historically, we have set this on deployment with deploystudio.  In the r8 and m1 world, we are managing the LocalHostName in MDM.  So that would conflict.  If r7s do not have LocalHostName set correctly, then we should fix this in deploystudio and then do a bolt command to fix all the existing nodes.

Reply:
Great! I didn't consider MDM and saw the install-created LocalHostName coming up in syslog for the R8's. I'll remove this.